<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit_recieve">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>33</y>
      <width>256</width>
      <height>173</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_send">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>256</y>
      <width>256</width>
      <height>173</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>接收窗口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>发送窗口</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_com">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>33</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>33</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>串口号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>61</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_baund">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>61</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>89</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数据位:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_data">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>89</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>117</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>校验位:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_check">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>117</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>145</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>停止位:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_stop">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>145</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="open_com">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>248</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_data">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>283</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>发送数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_data">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>318</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>清空数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="check_com">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>213</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>检测串口</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_delayedsending">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>定时发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="draw_pic">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>绘制图像</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>442</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
